pkg/eventhandler/eventhandlers.go

```diff
@@ -115,26 +115,32 @@ func (eh *EventHandler) HandleEvent() error {
 		return err
 	}
 
-	match, action := configuration.IsEventMatch(eh.Keptn.CloudEvent.Type(), eventAsInterface)
-	if !match {
+	// Check if we have an action that we can use, if it isn't the case we produce a log output
+	hasMatchingEvent := configuration.IsEventMatch(eh.Keptn.CloudEvent.Type(), eventAsInterface)
+	if !hasMatchingEvent {
 		log.Printf(
 			"No match found for event %s of type %s. Skipping...", eh.Keptn.CloudEvent.Context.GetID(),
 			eh.Keptn.CloudEvent.Type(),
 		)
 		return nil
 	}
 
-	log.Printf(
-		"Match found for event %s of type %s. Starting k8s job to run action '%s'", eh.Keptn.CloudEvent.Context.GetID(),
-		eh.Keptn.CloudEvent.Type(), action.Name,
-	)
+	// For each action that matches the given event type we execute all containing tasks:
+	for actionIndex, action := range configuration.Actions {
+		if action.IsEventMatch(eh.Keptn.CloudEvent.Type(), eventAsInterface) {
+			log.Printf(
+				"Match found for event %s of type %s. Starting k8s job to run action '%s'", eh.Keptn.CloudEvent.Context.GetID(),
+				eh.Keptn.CloudEvent.Type(), action.Name,
+			)
```

Comment:
I haven't checked the issue, but it might solve it. The change was more or less necessary to get the index of the targeted action without producing too hacky code.